package.json

```diff
@@ -68,6 +68,7 @@
     "proxy-polyfill": "0.1.6",
     "re-reselect": "^4.0.0",
```

Comment:
When editing playlists...
(although I vague recall from the meeting that is work-in-progress.  can dismiss if so)

![image](https://user-images.githubusercontent.com/64950861/150520567-69a911cb-3a47-4db0-ad07-48c07e81ebcd.png)

Reply:
Umm...they are greyed out. Not quite sure how you were able to get to this result, but this page looks horrible anyways. Gonna redesign it right now.
![image](https://user-images.githubusercontent.com/94839278/150524318-5495637a-dd97-44b1-87ce-012a2e9d8e3f.png)

Reply:
Oh, I happened to have abandoned claims at both ends, so not related to theme,
![image](https://user-images.githubusercontent.com/64950861/150524834-a4e9dbf8-15a4-4aa3-9332-36c973ed1d27.png)

Maybe some spacing here? But ok to leave as is if that is part of the new style.
https://odysee.com/$/list/0e99f72ecafd2fc3a1b873c3059e182176196365
![image](https://user-images.githubusercontent.com/64950861/150525060-a4c77b5a-03aa-423f-a098-93e7233d9cf7.png)

Reply:
The "Follow" location is also odd, but it's an existing issue with current Odysee too

![image](https://user-images.githubusercontent.com/64950861/150525345-d8e8e28f-0ed2-4fd1-b4a6-6516818dc0ca.png)

Reply:
Fixed.
![image](https://user-images.githubusercontent.com/94839278/150533454-d19c8fdb-cb92-4771-8cc5-9ae94f6a6795.png)

Reply:
Both issues are fixed, but the code is kinda messy. The playlists are currently using existing classes and I had to manipulate them for this specifis use case. I'm gonna write seperated classes for everything that involves playlists in the next release.
![image](https://user-images.githubusercontent.com/94839278/150568016-45ba3d92-62ba-4d7c-88f2-1b7627b50ea7.png)